compositor_render/src/transformations/text_renderer.rs

```diff
@@ -206,13 +209,15 @@ impl From<&TextComponent> for TextParams {
 pub struct TextRendererCtx {
     font_system: Mutex<FontSystem>,
     swash_cache: Mutex<SwashCache>,
+    cache: Mutex<Cache>,
 }
 
 impl TextRendererCtx {
-    pub(crate) fn new() -> Self {
+    pub(crate) fn new(device: &wgpu::Device) -> Self {
         Self {
             font_system: Mutex::new(FontSystem::new()),
             swash_cache: Mutex::new(SwashCache::new()),
+            cache: Mutex::new(Cache::new(device)),
```

Comment:
If those values are always used together, use one struct and one mutex and lock on it.

Reply:
theyre not always used together